### PR TITLE
Add display names feature

### LINKED ISSUE #84: Owner name in posts fields
Hey over here !

We started (again) working with your API, but one question remains for us : How could we retrieve the owner name of a post easily (`users.displayname` within the Bolt Schema) ?
`attributes` object already contains a `ownerid` field but it's a kinda pretty limited piece of information...

Thanks, bye :wave:

### PR DESCRIPTION
**Pull request by xiaohutai**
Requires opt-in via `enabledisplaynames` in config

I'm not 100% about adding a whole service for one feature.
Ideally this key, value is added before parse?

Fixes #84

### CODE REVIEW

File: src/Parser/Parser.php [hunk 1]
```diff
@@ -21,11 +22,15 @@ class Parser
     /** @var MetadataDriver $metadata */
     protected $metadata;
 
-    public function __construct(Config $config, ResourceManager $resourceManager, MetadataDriver $metadata)
+    /** @var Users $users */
+    protected $users;
+
+    public function __construct(Config $config, ResourceManager $resourceManager, MetadataDriver $metadata, Users $users)
```
Comment (GwendolenLynch) [inline src/Parser/Parser.php]: If BC is being observed, this is a break.

Comment (xiaohutai) [reply src/Parser/Parser.php]: Oh! Good point!

Thanks for checking @GawainLynch  👍

**Comment by xiaohutai** [comment]
Moved mandatory constructor param to public `setUsers`.

NB: Good resource on BC: https://symfony.com/doc/current/contributing/code/bc.html

**Comment by HorlogeSkynet** [comment]
Hey @xiaohutai ! Are we all good here ? Is this merge-able and bump-able to v4.0 ? :smiley:
Thanks, see you :wave:

**Comment by xiaohutai** [comment]
@HorlogeSkynet Watch out for `v3.3.0` soon ™️

**Comment by HorlogeSkynet** [comment]
Thanks for all, see you 👋🏻

**Comment by HorlogeSkynet** [comment]
Wow, such rude boys we are... Just wanted to tell you this DID PERFECTLY THE JOB. Thanks again. Bye :wave:

**Comment by xiaohutai** [comment]
😍
I appreciate your message. Glad you like it!

**Pull request merged**